src/modules/wishlists.js

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
Change the filename to something like wishlistsReducer so it's better distinguishable from other files